css/styles.css

```diff
@@ -201,20 +217,21 @@
 
 .ab-column--submit-service {
     width: 342px;
-    margin-top: 56px;
+    margin-top: 20px;
```

Comment:
There is arguably less space between the last option and the service-block now - the bigger gap made the service-box look oddly dispatched. But I can't find any alignment-problems. A screenshot of the misalignment would be great.
In `2.6.9` there was no special spacing between the service-box and the last option, this would clearly be a way nicer solution than mine with 20px. 🤔